components/vertex-components/pyproject.toml

```diff
@@ -1,32 +1,28 @@
-[project]
+[tool.poetry]
 name = "vertex-components"
-authors = [
-    {name = "Example User", email = "user@example.com"},
-]
-description = "Vertex AI components"
+version = "0.1.0"
+authors = ["Example User <user@example.com>"]
+description = "KFP components for interacting with Vertex AI"
 readme = "README.md"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3.7",
-]
-requires-python = ">=3.7"
-dynamic = ["version"]
-dependencies = [
-    "kfp == 1.8.21",
+    "Programming Language :: Python :: 3.9",
 ]
 
-[project.optional-dependencies]
-tests = [
-    "google-cloud-aiplatform == 1.24.1",
-    "google-cloud-pipeline-components == 1.0.42",
-    "pytest >= 7.3.1,<8.0.0",
-    "pre-commit >= 2.14.1,<3.0.0",
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<3.11"
+google-cloud-aiplatform = "1.24.1"
+google-cloud-pipeline-components = "1.0.42"
+kfp = "==1.8.21"
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=7.3.1,<8.0.0"
+pre-commit = ">=2.14.1,<3.0.0"
```

Comment:
```
[tool.poetry.dependencies]
python = ">=3.9,<3.11"
kfp = "==1.8.21"

[tool.poetry.group.dev.dependencies]
google-cloud-aiplatform = "1.24.1"
google-cloud-pipeline-components = "1.0.42"
pytest = ">=7.3.1,<8.0.0"
```

Reply:
cec9f3e

Reply:
Both bigquery and vertex components were squashed together in this commit